crates/astria-composer/src/executor/mod.rs

```diff
@@ -417,6 +431,35 @@ impl Executor {
 
         reason.map(|_| ())
     }
+
+    // check for mismatched configured chain_id and sequencer client chain_id
```

Comment:
This comment is wrong. You are not performing that check in here.

Also, if you want to add doc comments, use triple backticks: `/// <doc-string>`

Reply:
Fixed in 8523e04.